src/components/CommandBar/CommandBar.scss

```diff
@@ -61,7 +61,6 @@ $SearchBox-iconWrapperWidth: $SearchBox-widthLgCollapsed - ($SearchBox-sidePaddi
   display: inline;
   vertical-align: top;
   line-height: $CommandBar-height;
-  margin: 0 24px;
```

Comment:
I'm pretty sure that space is meant to be there. Why does this need to be removed?